src/lib/dnssd/minimal_mdns/ResponseSender.cpp

```diff
@@ -105,6 +107,12 @@ CHIP_ERROR ResponseSender::Respond(uint16_t messageId, const QueryData & query,
 {
     mSendState.Reset(messageId, query, querySource);
 
+    if (query.IsInternalBroadcast())
```

Comment:
Suggest rename `if (query.IsAnnounceBroadcast())`

Reply:
Updated since the name was unique enough for a search & replace.